lib/Sema/TypeCheckGeneric.cpp

```diff
@@ -618,8 +618,7 @@ static Type formExtensionInterfaceType(
   }
 
   // If we have a typealias, try to form type sugar.
-  if (typealias && TypeChecker::isPassThroughTypealias(
```

Comment:
Are there any remaining usages of TypeChecker::isPassThroughTypealias()?

Reply:
Doesn't look like it - Let's nuke it!